src/universal/modules/meeting/helpers/electFacilitatorIfNone.js

```diff
@@ -0,0 +1,29 @@
+import {cashay} from 'cashay';
+import {showInfo} from 'universal/modules/toast/ducks/toastDuck';
+
+const electFacilitatorIfNone = (nextProps, oldMembers) => {
```

Comment:
I think this function could use some comments. I am having a hard time grokking what each of the state checks imply semantically. I'll give some examples: